docs/components/content/examples/PaginationExampleFirstLastSlots.vue

```diff
@@ -5,15 +5,29 @@ const items = ref(Array(55))
 
 <template>
   <UPagination v-model="page" :total="items.length" :ui="{ rounded: 'first-of-type:rounded-s-md last-of-type:rounded-e-md' }">
-    <template #first="{ onClick }">
+    <template #first="{ onClick, canGoFirst }">
       <UTooltip text="First page">
-        <UButton icon="i-heroicons-arrow-uturn-left" color="primary" :ui="{ rounded: 'rounded-full' }" class="rtl:[&_span:first-child]:rotate-180 me-2" @click="onClick" />
+        <UButton
+          icon="i-heroicons-arrow-uturn-left"
+          color="primary"
+          :ui="{ rounded: 'rounded-full' }"
+          class="rtl:[&_span:first-child]:rotate-180 me-2"
+          :disabled="!canGoFirst"
+          @click="onClick"
+        />
```

Comment:
I wanted to pass the new prop into the existing line, but vue/max-attributes-per-line eslint rule threw error. So I made new lines